### DISCUSSION TITLE
What's difference of initialising connection between sunshine and parsec?

### DISCUSSION THREAD

**Discussion by Kobain-Seo** [LizardByte/community#347] [Sunshine] [answered]
I am not sure it is about issue of moonlight, but it takes multiple times longer to connect to host than parsec.
What's difference? The way of implement desktop environment?

**Comment by ReenigneArcher** [comment, marked as answer]
We probe the encoders, displays, etc. before each connection to ensure they are still available at the time of streaming versus when Sunshine is first started.

**Reply by Kobain-Seo** [reply]
Through the log, I assume Sunshine probes all encoders and displays. Especially frame rate is set to 60fps at the first time and it changes to target frame rate. Can I force such of progress by configuration?
